packages/tables/src/elements/Row.spec.tsx

```diff
@@ -176,4 +176,35 @@ describe('Row', () => {
       expect(getByTestId('row')).toHaveStyleRule('height', '64px');
     });
   });
+
+  describe('ReadOnly', () => {
+    it('removes tabIndex while in read-only mode', () => {
+      const { getByTestId } = render(
+        <Table isReadOnly>
+          <Body>
+            <Row data-test-id="row" />
+          </Body>
+        </Table>
+      );
+
+      expect(getByTestId('row')).not.toHaveAttribute('tabindex');
+    });
+
+    it('removes :hover styling while in read-only mode', () => {
```

Comment:
A :hover styling is never removed in this test (e.g. it never appears because the table is read-only). What do you think about rephrasing this description to something along the lines of:

> does not show :hover styling while in read-only mode